release-notes/opensearch-anomaly-detection-dashboards.release-notes-2.15.0.0.md

```diff
@@ -0,0 +1,21 @@
+## Version 2.15.0.0 Release Notes
+
+Compatible with OpenSearch Dashboards 2.15.0
+
+### Enhancements
+
+* support MDS in feature anywhere ([#767](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/767))
+* update create detector page and detector detail page to add custom result index lifecycle management settings ([#770](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/770))
+* Update Frontend for Custom Result Index Query and Fix Issues ([#772](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/772))
+* make custom result index lifecycle management in AD optional ([#777](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/777))
+
+### Bug Fixes
+
+* Fix handling of special characters in categorical values ([#757](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/757))
+* Fix Warning Message About Custom Result Index Despite Existing Indices ([#759](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/759))
+* fix a MDS related bug on feature anywhere page ([#773](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/773))
```

Comment:
we can remove this, it's a bug fix for the line 7 item

Reply:
removed